Update PRIVACY.md (#169)
returntocorp/bento: PRIVACY.md

# Bento Privacy Policy

Bento collects usage data to help us improve the product. This document describes:

> * the principles that guide our data collection decisions

Reviewer: ```suggestion
* the principles that guide our data-collection decisions
```

> * the breakdown of the data that is and is not collected

Reviewer: ```suggestion
* the breakdown of the data that are and are not collected
```

I know, I spent too much time in the sciences...  One datum, two data.

> * how we use data we collect to make Bento better

Reviewer: ```suggestion
* how we use the data we collect to make Bento better
```

## Principles

These principles inform our decisions around data collection:

1. **Transparency**: Collect and use data in a way that is clearly explained to the user and benefits them
2. **User control**: Put users in control of their data at all times
3. **Limited data**: Collect what is needed, de-identify where possible, and delete when no longer necessary

## Collected Data

Bento collects data to help improve its underlying tools and user experience. Two types of data are collected:

### Usage Data

> As the name suggests, usage data tells us what commands you ran, what options and flags you ran them with, as well as information that helps us debug any issues users may be facing, e.g.

Reviewer: ```suggestion
As the name suggests, usage data tell us what commands you ran, what options and flags you ran them with, as well as information that helps us debug any issues users may be facing, e.g.
```

* The name of the command you ran,
* Arguments or options,
* How long the command took to run,
* If the command ran in a CI environment,
* The version of Bento you're using, 
* What OS and shell you're running in, etc.

See Description of Fields for a full list.

> While using Bento in Beta, we associate this data with your email address to facilitate our outreach efforts.

Reviewer: ```suggestion
While using Bento in Beta, we associate these data with your email address to facilitate our outreach efforts.
```

### Results Data

> Results data contains information about the code check rules that fired when you ran Bento, and helps us make our rules better. This includes:

Reviewer: ```suggestion
Results data contain information about the code check rules that fired when you ran Bento, and help us make our rules better. These include:
```

> * Data on the number of times rules fired across your repo,

Reviewer: ```suggestion
* The number of times rules fired across your repo,
```

* Rules you disabled,
* The number of results that were fixed or archived since the last run, and

> * Hashes of the repository and commit Bento produced these results for,

Reviewer: ```suggestion
* Hashes of the repository and commit for which Bento produced these results,
```

> In addition, we collect a hashed version of the repo and commit Bento was run on. We consider this information more sensitive and do NOT associate it with user-identifiable information.

Reviewer: The antecedent of "this information" is confusing. Do you mean that we consider the repo hash to be sensitive, or do we consider the results data to be sensitive?

### Data NOT collected

> We try hard to balance our desire to collect data for improving Bento and its underlying tools with our users' need for privacy. It does not leave your computer and is not sent or shared with anyone.

Reviewer: "try hard" seems weird to me. What about "We strive"?

Reviewer: What is the "It" that does not leave your computer?

Author: woops
